federation/backends/minimal_profile.md

```diff
@@ -0,0 +1,175 @@
+    },
+    {
+      "methods": [
+        "GET",
+        "DELETE"
+      ],
+      "path": "/jobs/{job_id}"
+    },
+    {
+      "methods": [
+        "GET",
+        "POST",
+        "DELETE"
+      ],
+      "path": "/jobs/{job_id}/results"
+    }
+  ],
+```
+
+## Batch job support
+
+Support for openEO batch jobs is mandatory, but certain specific elements can be ommitted. The previous section indicates the exact endpoints and HTTP requests that need to be supported.
+
+
+## File formats
+[GeoTiff](./fileformats.md#geotiff) is the only mandatory output format.
+
+## Collections
+A minimal backend shall have at least one raster collection, allowing to build raster data cubes. 
+
+## Processes
+
+### Data Cubes
+
+- `apply`: Apply a process to each pixel
+- `apply_dimension`: Apply a process to pixels along a dimension
+- `dimension_labels`: Get the dimension labels
+- `filter_bands`: Filter the bands by names
+- `filter_bbox`: Spatial filter using a bounding box
+- `filter_spatial`: Spatial filter using geometries
+- `filter_temporal`: Temporal filter for temporal intervals
+- `load_collection`: Load a collection
+- `mask`: Apply a raster mask
+- `resample_spatial`: Resample and warp the spatial dimensions
+- `save_result`: Save processed data
+
+For backends with more than one collection:
+- `merge_cubes`: Merge two data cubes
+- `resample_cube_spatial`: Resample the spatial dimensions to match a target data cube
+
+
+### Math
+- `absolute`: Absolute value
+- `add`: Addition of two numbers
+- `arccos`: Inverse cosine
+- `arcosh`: Inverse hyperbolic cosine
+- `arcsin`: Inverse sine
+- `arctan`: Inverse tangent
+- `arctan2`: Inverse tangent of two numbers
+- `arsinh`: Inverse hyperbolic sine
+- `artanh`: Inverse hyperbolic tangent
+- `ceil`: Round fractions up
+- `clip`: Clip a value between a minimum and a maximum
+- `constant`: Define a constant value -> pretty easy implementation
+- `cos`: Cosine
+- `cosh`: Hyperbolic cosine
+- `divide`: Division of two numbers
+- `e`: Euler's number
+- `exp`: Exponentiation to the base e
+- `floor`: Round fractions down
+- `int`: Integer part of a number
+- `linear_scale_range`: Linear transformation between two ranges
+- `ln`: Natural logarithm
+- `log`: Logarithm to a base
+- `mod`: Modulo
+- `multiply`: Multiplication of two numbers
+- `pi`: Pi
+- `power`: Exponentiation
+- `round`: Round to a specified precision
+- `sgn`: Signum
+- `sin`: Sine
+- `sinh`: Hyperbolic sine
+- `sqrt`: Square root
+- `subtract`: Subtraction of two numbers
+- `tan`: Tangent
+- `tanh`: Hyperbolic tangent
+
+### Indices
+- `normalized_difference`: Normalized difference
+
+### Logic
+- `and`: Logical AND
+- `all`: Are all of the values true?
+- `any`: Is at least one value true?
+- `between`: Between comparison
+- `eq`: Equal to comparison
+- `gt`: Greater than comparison
+- `gte`: Greater than or equal to comparison
+- `if`: If-Then-Else conditional
+- `is_infinite`: Value is an infinite number - **experimental**
```

Comment:
I would not list experimental processes as required in a minimal_profile.

Reply:
Indeed, it is also not implemented yet by any backend afaik. I thought backends would implement it soon when we added it to the other profile as it's a relatively simple process, but that did not happen yet. So I think we can remove it for now (or start implementing it).